contracts/core/Dispatcher.sol

```diff
@@ -173,24 +135,24 @@ contract Dispatcher is IbcDispatcher, IbcEventsEmitter, Ownable, Ibc {
         Ics23Proof calldata proof
     ) external {
         if (bytes(counterparty.portId).length == 0) {
-            revert invalidCounterPartyPortId();
+            revert IBCErrors.invalidCounterPartyPortId();
         }
 
-        if (isChannelOpenTry(counterparty)) {
-            consensusStateManager.verifyMembership(
+        if (IbcUtils.isChannelOpenTry(counterparty)) {
```

Comment:
I am currently working on an API clean up that will remove this method and change the channel handshake api a bit. Maybe leave it here instead of moving it to a library to avoid bigger conflicts?